libs/api/src/lib/graphql/comment-rating/comment-rating.public-mutation.ts

```diff
@@ -70,4 +73,16 @@ export const rateComment = async (
       fingerprint
     }
   })
+
+  const comment = await commentClient.findFirst({
```

Comment:
Returning the comment always the frontend to leverage caching instead of having to add logic for the first time a user rates a specific comment 8 answer.